app/Filament/Resources/GameResource.php

```diff
@@ -189,6 +201,57 @@ public static function form(Form $form): Form
                             ->suffixIcon('heroicon-m-globe-alt')
                             ->disabled(!$user->can('updateField', [$form->model, 'GuideURL'])),
                     ]),
+
+                Forms\Components\Section::make('Earliest Release Date')
+                    ->icon('heroicon-c-calendar-days')
+                    ->description("
+                        Provide the game's earliest known release date to improve searching, sorting, and filtering on the site.
+                        Use the Precision control to specify if the release day or month are unknown.
+                    ")
+                    ->columns(['md' => 2, 'xl' => 3, '2xl' => 4])
+                    ->schema([
+                        Forms\Components\Placeholder::make('released_at_display')
+                            ->label('Preview')
```

Comment:
Originally, I wanted to dynamically adjust the datepicker's `displayFormat()` based on the user's selected granularity choice.

However, it seems the datepicker control doesn't support this functionality. The complexity of building a custom control for this use case is relatively high. This seems like the happy medium.

Reply:
Should the display (non-edit page) show the granular (preview) value?
![image](https://github.com/user-attachments/assets/4c9e40eb-07f1-4898-b3ed-7f22c012b04b)

Reply:
Release Date and Release Date Precision don't appear in the audit log.

Reply:
Good points - should both be fixed in latest.